### PR TITLE
Improve bigfiles situation on mysql branch

### PR DESCRIPTION
**Pull request by vinceatbluelabs**

### CODE REVIEW

File: tests/integration/records/table_validator.py [hunk 1]
```diff
@@ -17,18 +17,7 @@
 logger = logging.getLogger(__name__)
 
 
-#
```
Comment (bluelabsbutler) [inline tests/integration/records/table_validator.py:20]: This file, one of the 3 largest in the project, increased in size to 462 lines.  The total size of those files is now 3504 lines (target: 897).  Is this file complex enough to refactor?

File: tests/integration/records/table_validator.py [hunk 2]
```diff
@@ -12,6 +12,7 @@
     expected_table2table_column_types
 )
 from records_mover.records import DelimitedVariant
+from .mover_test_case import MoverTestCase
```
Comment (bluelabsbutler) [inline tests/integration/records/table_validator.py:15]: This file, one of the 3 largest in the project, increased in size to 398 lines.  The total size of those files is now 3440 lines (target: 897).  Is this file complex enough to refactor?

File: tests/integration/records/table_validator.py [hunk 3]
```diff
@@ -12,6 +12,7 @@
...
 
 
 logger = logging.getLogger(__name__)
```
Comment (bluelabsbutler) [inline tests/integration/records/table_validator.py:18]: F401 'typing.List' imported but unused

File: tests/integration/records/table_validator.py [hunk 4]
```diff
@@ -294,15 +201,15 @@ def validate_data_values(self,
                 f"ret['timestamp'] was {ret['timestamp']} of type {type(ret['timestamp'])}"
```
Comment (bluelabsbutler) [inline tests/integration/records/table_validator.py:202]: C901 'RecordsTableValidator.validate_data_values' is too complex (17)

File: metrics/bigfiles_high_water_mark [hunk 5]
```diff
@@ -1 +1 @@
-897
+967
```
Comment (vinceatbluelabs) [inline metrics/bigfiles_high_water_mark:1]: After this, `table_validator.py` and `types/stubs/logging/__init__.pyi` are off of the list.  The MySQL additions to `setup.py` and `itest` are still hanging around, though - I'm proposing to accept that increase for now and take on refactoring those two pieces as part of their respective backlog items.

```
350: itest
317: records_mover/records/schema/field/__init__.py
300: setup.py

Reduce total number of bigfiles violations to 897 or below!

Tasks: TOP => bigfiles
(See full trace by running task with --trace)
Existing violations: 897
Found 967 bigfiles violations
```

**Comment by vinceatbluelabs** [comment]
Hi @cwegrzyn !  Could you review when you get a chance?

**Comment by vinceatbluelabs** [comment]
Hi @cwegrzyn!  If you get a chance, reviewing this one would unblock a lot of additional work.

**Pull request merged**